ArtifactOntology.ttl

```diff
@@ -4750,7 +4751,7 @@ cco:TurbojetAirBreathingJetEngine rdf:type owl:Class ;
 ###  http://www.ontologyrepository.com/CommonCoreOntologies/TwoDimensionalBarCode
 cco:TwoDimensionalBarCode rdf:type owl:Class ;
                           rdfs:subClassOf cco:Barcode ;
-                          cco:definition "A Barcode whose concretizations consist of two-dimensional symbol patterns."@en ;
+                          cco:definition "A Barcode that consists of a two-dimensional symbol pattern."@en ;
```

Comment:
Suggested revisions to parent and subclasses:
- Barcode - An Information Bearing Artifact that is designed to bear one or more geometric shapes that concretize some Directive Information Content Entity.

- One-Dimensional Barcode - A Barcode that is designed to bear parallel lines of varying widths and spacing that concretize some Directive Information Content Entity.

- Two-Dimensional Barcode - A Barcode that is designed to bear lines of varying widths, spacing, height, and color that concretize some Directive Information Content Entity.